staging/src/k8s.io/client-go/pkg/apis/clientauthentication/v1alpha1/zz_generated.conversion.go

```diff
@@ -119,14 +119,10 @@ func Convert_v1alpha1_ExecCredentialSpec_To_clientauthentication_ExecCredentialS
 func autoConvert_clientauthentication_ExecCredentialSpec_To_v1alpha1_ExecCredentialSpec(in *clientauthentication.ExecCredentialSpec, out *ExecCredentialSpec, s conversion.Scope) error {
 	out.Response = (*Response)(unsafe.Pointer(in.Response))
 	out.Interactive = in.Interactive
+	// WARNING: in.Cluster requires manual conversion: does not exist in peer-type
```

Comment:
was this field accidentally not added to both types?

Reply:
can add `// +k8s:conversion-gen=false` to the field if this was intentionally not converted

Reply:
Yeah I was using it as a forcing function to "you must start using the v1beta1 API if you want this field."